apps/docs/styles/tokens.css

```diff
@@ -48,6 +42,7 @@
     --hd-space-8: 4rem;
     --hd-space-9: 4.5rem;
     --hd-space-10: 5rem;
+    --hd-border-size: 1px; /* stylelint-disable-line unit-allowed-list */
```

Comment:
is it the only border size we expect to have? If not we could have a scale I guess.